sellercenter-filemanger/pom.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.gfg.sellercenter.filemanger</groupId>
+    <artifactId>sellercenter-filemanger</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <name>sellercenter-filemanger</name>
+    <url>https://github.com/GFG/java-packages</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <java.version>11</java.version>
```

Comment:
I never had a time to read how actually jar files compatible to each other. So if someone is using java 8, will it be possible to include java 11 jar library?